requirements/base.txt

```diff
@@ -4,6 +4,7 @@ djangorestframework==3.7.7
 django==1.11.10  # pyup: >=1.11,<2
 six==1.11.0
 colorlog==3.1.2
+configobj==5.0.6
```

Comment:
Initially, in the .ini proposal, it was partly also because we wouldn't have to add dependencies IIRC :)

Reply:
The problem was that `configparser` isn't able to read/write config files while preserving order, comments, etc, while `configobj` can (as well as having nicer validation logic, etc).

We want to be able to write out options.ini templates that include comments, and modify existing files to update the content directory setting for @lyw07's PR.